frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.module.css

```diff
@@ -62,3 +62,7 @@
   right: 0;
   bottom: 0;
 }
+
+.handle {
+  top: auto;
+}
```

Comment:
The default value of the top property was 50%. I've overridden this to a new value.